h2o-py/h2o/utils/shared_utils.py

```diff
@@ -162,43 +162,6 @@ def _gen_header(cols):
     return ["C" + str(c) for c in range(1, cols + 1, 1)]
 
 
-def _check_lists_of_lists(python_obj):
```

Comment:
just moved those functions with import business logic to `frame.py` where it should be.
`shared_utils` should ideally only contain simple utility functions can be used anywhere, nothing too specific to a business module, otherwise it's messy.